### DISCUSSION TITLE
Multi Search and select all checkbox

### DISCUSSION THREAD

**Discussion by sandebains** [sa-si-dev/virtual-select#158] [General]
On a large dropdown list; if the options selection require multiple searches, the dropdown only shows the latest search options if it is selected via "Select All" checkbox. Using the "Select all" checkbox causes the earlier selected option to be de-selected. Is there an option or if it can be added where it retains the existing options and add the newly searched options on pressing "Select all" checkbox?

**Comment by sa-si-dev** [comment]
@sandebains there is no feature like this as of now. I will implement it in the future release.
